src/json/brace-data-exchange.schema.json

```diff
@@ -528,9 +532,9 @@
           "description": "The date of the last Application Status change"
         },
         "sor_application_date": {
+          "$ref": "#/definitions/mdy_date",
```

Comment:
Is this the _only_ format we want to accept for this field? Or should we change the regex / use an `anyOf` or something fun like that?

Reply:
only this one client uses this field, at the present time, and they are the only client using this custom format. we are locking down the format for dates going forward.